app/components/hint.tsx

```diff
@@ -338,6 +338,6 @@ const mkHint = (
 };
 
 export const HintRed = mkHint("red", "HintRed");
-export const HintGreen = mkHint("green", "HintGreen");
+const HintGreen = mkHint("green", "HintGreen");
```

Comment:
I see this is not exported anymore, don't we use a Hint green when the chart has been published ?

Reply:
Yep, the thing is that it's also defined in this file and knip was complaining about the export 😅 

```ts
export const PublishSuccess = () => {
  return (
    <HintGreen>
      <Trans id="hint.publication.success">
        Your visualization is now published. You can share and embed it using
        the URL or the options below.
      </Trans>
    </HintGreen>
  );
};
```